packages/examples/src/person.ts

```diff
@@ -79,22 +79,8 @@ export const uischema = {
         },
         {
           type: 'Control',
-          label: '%height',
-          scope: '#/properties/personalData/properties/height'
-        },
-        {
-          type: 'Control',
-          label: {
-            text: '%age'
-          },
-          scope: '#/properties/personalData/properties/age'
-        },
-        {
-          type: 'Control',
```

Comment:
we should at least move the driving skill example to an own example eg "numbers" to demonstrate different number behaviors